rapidtide/correlate.py

```diff
@@ -567,15 +585,15 @@ def delayedcorr(data1, data2, delayval, timestep):
 
     Returns
     -------
-
+    corr
     """
     return sp.stats.stats.pearsonr(
         data1, tide_resample.timeshift(data2, delayval / timestep, 30)[0]
     )
 
 
 def cepstraldelay(data1, data2, timestep, displayplots=True):
-    """
+    """Calculate something.
```

Comment:
I don't know what this one does 😞

Reply:
I've updated the description and moved the reference up where it's visible.